### ISSUE TITLE
希望可以修改状态栏的设备拥有者名称

### ISSUE THREAD

**Issue by CuiHM** [iamr0s/Dhizuku#43]
现在状态栏默认显示“此设备归贵单位所有”，希望可以自定义修改“贵单位”这个词条，这样可以改为用户自己的姓名！

**Comment by CuiHM** [comment]
并且希望可以适配Islan（炼妖壶），可以让Dhiziku授权激活Islan（炼妖壶）

**Comment by ManyullynSword** [comment]
> 现在状态栏默认显示“此设备归贵单位所有”，希望可以自定义修改“贵单位”这个词条，这样可以改为用户自己的姓名！

可以做到，用 Dhizuku 授权爱玩机工具箱或安卓管理员这两个应用其一，它们都能修改。设置修改不是 Dhizuku 这个权限共享器要内置的。

**Comment by LegendEast** [comment]
DeviceOwner权限理论上不可以和工作空间共存

**Comment by 556gfhjnddf** [comment]
这个好像用爱玩机可以修改哦

**Comment by chumoew** [comment]
可以使用OwnDroid 6.0更改组织名称。首先下载 https://github.com/BinTianqi/OwnDroid/releases，进入OwnDroid点击未激活。跳转到权限界面后勾选Dhizuku并同意权限申请即可完成激活。在OwnDroid主界面点击已激活，再次进入权限界面后点击第六行的组织名称即可自定义更改，但只能更改双引号内的文字。

**Comment by 556gfhjnddf** [comment]
懂了。



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年8月28日(周三) 上午10:39
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [iamr0s/Dhizuku] 希望可以修改状态栏的设备拥有者名称 (Issue #43)




 
可以使用OwnDroid 6.0更改组织名称。首先下载 https://github.com/BinTianqi/OwnDroid/releases，进入OwnDroid点击未激活。跳转到权限界面后勾选Dhizuku并同意权限申请即可完成激活。在OwnDroid主界面点击已激活，再次进入权限界面后点击第六行的组织名称即可自定义更改，但只能更改双引号内的文字。

**Comment by Johngithubandroidpc** [comment]
Is there a way to use an Android app to grant device owner privileges, like Icebox, so I don't have to connect my Android 10 phone to a computer for ADB root access? Thank you!

**Comment by LegendEast** [comment]
This can be achieved by transferring the permissions of OwnDroid to Dhizuku, then enabling Dhizuku mode in OwnDroid, and finally applying the DAX patch to IceBox.

**Comment by Johngithubandroidpc** [comment]
> This can be achieved by transferring the permissions of OwnDroid to Dhizuku, then enabling Dhizuku mode in OwnDroid, and finally applying the DAX patch to IceBox.

Could you clarify if a factory reset is necessary to activate Owndroid in device owner mode before transferring privileges to Icebox? Or is it possible to enable Owndroid’s device owner mode without resetting, then transfer the privilege to Icebox? I’m currently using Shizuku instead of Dizuku, but it seems that after activating Owndroid, it gets deactivated again when I reopen it from Icebox. Would switching to Dizuku instead of Shizuku help? Thank you LegendEast!

**Comment by Johngithubandroidpc** [comment]
> This can be achieved by transferring the permissions of OwnDroid to Dhizuku, then enabling Dhizuku mode in OwnDroid, and finally applying the DAX patch to IceBox.

Where can I find the DAX patch for Icebox? Thanks!

**Comment by Johngithubandroidpc** [comment]
> This can be achieved by transferring the permissions of OwnDroid to Dhizuku, then enabling Dhizuku mode in OwnDroid, and finally applying the DAX patch to IceBox.

In short, I finally managed to get both Owndroid and Dhizuku working as device owners. Could you let me know where I can find the DAX patch for Icebox? Thanks!

**Comment by LegendEast** [comment]
> > This can be achieved by transferring the permissions of OwnDroid to Dhizuku, then enabling Dhizuku mode in OwnDroid, and finally applying the DAX patch to IceBox.
> 
> In short, I finally managed to get both Owndroid and Dhizuku working as device owners. Could you let me know where I can find the DAX patch for Icebox? Thanks!

LSPatch，a GitHub Project

**Comment by 556gfhjnddf** [comment]
我不知道哎，你可以通过ai搜索或各种渠道寻找
I don't know. You can search through ai or various channels.
わかりませんが、ai検索やいろいろなルートで探すことができます。
Je ne sais pas hein, tu peux chercher par ai search ou par divers canaux












珏印
***@***.***






---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年11月1日(周五) 晚上6:22
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [iamr0s/Dhizuku] 希望可以修改状态栏的设备拥有者名称 (Issue #43)




   
This can be achieved by transferring the permissions of OwnDroid to Dhizuku, then enabling Dhizuku mode in OwnDroid, and finally applying the DAX patch to IceBox.
  
In short, I finally managed to get both Owndroid and Dhizuku working as device owners. Could you let me know where I can find the DAX patch for Icebox? Thanks!
  
LSPatch，a GitHub Project

**Comment by Johngithubandroidpc** [comment]
> > > This can be achieved by transferring the permissions of OwnDroid to Dhizuku, then enabling Dhizuku mode in OwnDroid, and finally applying the DAX patch to IceBox.
> > 
> > In short, I finally managed to get both Owndroid and Dhizuku working as device owners. Could you let me know where I can find the DAX patch for Icebox? Thanks!
> 
> LSPatch，a GitHub Project
> 

Hello, I'd like to know how to safely uninstall both the Shizuku and OwnDroid apps, and reinstall them afterwards. Currently, I'm facing constant pop-ups with "com.android.phone has stopped" and "com.android.system.UI has stopped," and I need to resolve this issue. Thank you.

**Comment by 556gfhjnddf** [comment]
实在很抱歉，我并不完全理解您的意思，您可以再说的详细一点吗？


I'm really sorry, but I don't quite understand what you mean. Could you be more specific?


谢谢！
Thank you!












珏印
***@***.***






---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年11月3日(周日) 晚上11:06
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [iamr0s/Dhizuku] 希望可以修改状态栏的设备拥有者名称 (Issue #43)




    
This can be achieved by transferring the permissions of OwnDroid to Dhizuku, then enabling Dhizuku mode in OwnDroid, and finally applying the DAX patch to IceBox.
  
In short, I finally managed to get both Owndroid and Dhizuku working as device owners. Could you let me know where I can find the DAX patch for Icebox? Thanks!
  
LSPatch，a GitHub Project
  
Hello, I'd like to know how to safely uninstall both the Shizuku and OwnDroid apps, and reinstall them afterwards. Currently, I'm facing constant pop-ups with "com.android.phone has stopped" and "com.android.system.UI has stopped," and I need to resolve this issue. Thank you.
